ui_test/POM_hiway_asg/test_scripts/conftest.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+from selenium import webdriver
+from POM_hiway_asg.pages.constant_var import *
+
+
+@pytest.yield_fixture
+def setUp():
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename function "setUp" to match the regular expression ^[a-z_][a-z0-9_]{2,}$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=python%3AS1542)